sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -122,12 +124,24 @@
  * store) This requires each resource to contain a client provided ID. It is important that when
  * using import you give the appropriate permissions to the Google Cloud Healthcare Service Agent.
  *
+ * <p>ExportGcs This is to export FHIR resources from a FHIR store to Google Cloud Storage. The
```

Comment:
I think we should rename this to Export since GCS is an implementation detail here. Feel free to leave it to a follow-up PR.

Reply:
Good point. The reason I want to keep this as ExportGcs is there is also an ExportBQ. I agree that both are implementation details, but depending on usage, users may want a workflow connect to BQ. So I prefer to keep it this way for future extension.

Reply:
I see your point. That makes sense for this implementation because we will read the resources back anyway. However, export FHIR resources to BigQuery allows users to specify different schema, which could be non-trivial for BigqueryIO to reproduce.

I made the change anyway and if there is a need like above, we can add the ExportBQ later.

Reply:
I think I am missing something :) Let's talk offline.

Reply:
Talked offline, one use case Truc was thinking about is that users might want to use the ExportResources API to flatten the FHIR resources first, then load the flatten resources into the pipeline.

Reply:
Resolved offline.